src/test/java/com/microsoft/cse/helium/app/HeliumJavaApplicationTests.java

```diff
@@ -0,0 +1,15 @@
+/*
```

Comment:
If you are inserting a unit test in another file, feel free to remove this one. This is just a sample and doesn't have much meaning.

Reply:
Removed.